ArduCopter/Copter.h

```diff
@@ -348,6 +348,14 @@ class Copter : public AP_Vehicle {
         uint32_t clear_ms;  // system time high vibrations stopped
     } vibration_check;
 
+    // EKF variances are unfiltered and are designed to recover very quickly when possible
+    // thus failsafes should be triggered on filtered values in order to avoid transient errors 
+    LowPassFilterFloat position_variance;
```

Comment:
just a little naming thing, I'd probably have suffixed these with "_filt" so perhaps pos_var_filt or something like that.

Reply:
Fixed